### ISSUE TITLE
KeyboardHelpDialog is broken

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#431]
Just to mark it as a problem, it will be worked on as part of #424 

Tagging @jessegreenberg and @pixelzoom so they are aware.

 JT's keyboard dialog is currently failing in fuzz testing.
```
Uncaught Error: Assertion failed: This Node has already been disposed, and cannot be disposed again
Error: Assertion failed: This Node has already been disposed, and cannot be disposed again
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/assert/js/assert.js:21:13)
    at Node.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/nodes/Node.js?bust=1499727291801:502:19)
    at KeyboardHelpDialog.Dialog.disposeDialog (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/joist/js/Dialog.js?bust=1499727291801:226:21)
    at KeyboardHelpDialog.hide (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/joist/js/Dialog.js?bust=1499727291801:285:14)
    at Object.fire (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery-phet/js/BarrierRectangle.js?bust=1499727291801:48:57)
    at ButtonListener.setButtonState (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/input/ButtonListener.js?bust=1499727291801:129:31)
    at Object.up (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/input/ButtonListener.js?bust=1499727291801:87:14)
    at ButtonListener.buttonUp (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/input/DownUpListener.js?bust=1499727291801:125:22)
    at Object.up (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/input/DownUpListener.js?bust=1499727291801:58:16)
    at Input.dispatchToPointer (https://bayes.colorado.edu/continuous-testing/snapshot-1499725450369/scenery/js/input/Input.js?bust=1499727291801:775:27)
Approximately 7/10/2017, 2:24:10 PM

**Comment by jessegreenberg** [comment]
Thanks @zepumph, yes I am aware and it should be fixed once #424 is addressed.

**Comment by jessegreenberg** [comment]
#424 is still open but since hide and dispose are decoupled, the persistent KeyboardHelpDialog is working again.
